Fatal error: exception Failure("duplicate global i")